### ISSUE TITLE
How it possible to build xrGame on Windows 7 x64?

### ISSUE THREAD

**Issue by lekzd** [xrOxygen/xray-oxygen#211]
What i tried:
- VS2015 (issues with newest C++ features, can not build)
- VS2017 build was successful, but I can not run executable, because of target Windows SDK version is "10" or "8.1", and no able to choose "7" (similar to x64 and win32 configurations).

How to deal with it? Because of released version from github releases page works well on Windows 7 

I just want to switch on AF_PSP actor flag using console, seems like it only one more string to all code.

**Comment by Giperionn** [comment]
День добрый.
Большинство разработчиков сидит на Windows 10, поэтому мы даже и незнали об этом.
Мы используем последний SDK чтобы использовать все новые плюшки когда они только только выходят. Поэтому вот так.

**Comment by ForserX** [comment]
Отправь свою правку к нам через PR, я скину тебе собранный вариант, аналогичный готовым релизам.

**Comment by lekzd** [comment]
@ForserX надеюсь, оно работает, как надо:
https://github.com/xrOxygen/xray-oxygen/pull/212

**Comment by lekzd** [comment]
взял артефакт с appveyor, там нормально собираются бинарники, думаю на первое время этого хватит. Фича почему-то не работает, видно надо еще поинвестигировать, как это работает

**Comment by MatthewKush** [comment]
@lekzd I'd use Google Translate to understand the Russian responses, but I (currently) only know English, I'm in a hurry, and I suspect I know what the issue is.

xrRender/xrRenderPC_R4/stdafx.h is the culprit, and it's due to the DXGI version specified in the file: it's only in the Win10 SDK. I'm assuming it's this because I remember posting this code, not thinking people might use it before I was finished, which is exactly what happened. But that's my fault for a variety of reasons.

tl;dr: Check the file I named, check the includes for "dxgi_whatever_version" and make sure that the version it lists is also available in Windows 7. I already know the answer is "no" but I suggest reading up on it instead of taking me at my word.

**Comment by lekzd** [comment]
@MatthewKush I have problems with XAudio2 library, that wants Win10 SDK version only. And I was shocked about builds from appveyor works on my mashine.

**Comment by ForserX** [comment]
I don't have win10. Use ox_dev brach. xAudion on ox_dev maybe support Win7
